### PR TITLE
[Nemanager] Migrate analytics apis

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- Migrate apis to v2
- Remove unused urls and their config
- Pending: device monitoring, data management, locate, predict(backend pending)

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [AN-512](https://airqoteam.atlassian.net/browse/AN-512) 

#### Screenshots (optional)

[AN-512]: https://airqoteam.atlassian.net/browse/AN-512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Codebmk** [comment]
@Baalmart

1. Regarding the pending changes, it appears that the maintainers have not migrated to v2 yet. Consequently, attempting to access routes with just v2 added will result in a 404 (Not Found) or server errors.
2. The current changes primarily involve route configurations and not feature modifications. Therefore, these changes do not require tests.
3. To ensure thorough testing, I have manually reviewed all the features, especially those involving button clicks. Additionally, I have shared my feedback on items that require backend fixes. Documentation on the **changelog** for v2 routes would be valuable for tracking specific changes effectively.
4. For deploy, recall, and maintenance routes, I have left them as v1 for now. Daniel is currently working on these and will be utilizing v2.

Thanks

**Comment by Baalmart** [comment]
thanks @Codebmk , all is good now!

> @Baalmart
> 
> 1. Regarding the pending changes, it appears that the maintainers have not migrated to v2 yet. Consequently, attempting to access routes with just v2 added will result in a 404 (Not Found) or server errors.
> 2. The current changes primarily involve route configurations and not feature modifications. Therefore, these changes do not require tests.
> 3. To ensure thorough testing, I have manually reviewed all the features, especially those involving button clicks. Additionally, I have shared my feedback on items that require backend fixes. Documentation on the **changelog** for v2 routes would be valuable for tracking specific changes effectively.
> 4. For deploy, recall, and maintenance routes, I have left them as v1 for now. Daniel is currently working on these and will be utilizing v2.
> 
> Thanks

**Review by Baalmart — APPROVED** [review]

File: netmanager/src/config/urls/deviceRegistry.js [hunk 1]
```diff
@@ -1,59 +1,57 @@
...
+export const DEPLOY_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/deploy`; // Change to V2
 
-export const ADD_MAINTENANCE_LOGS_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/maintain`;
+export const ADD_MAINTENANCE_LOGS_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/maintain`; // Change to V2
 
-export const DELETE_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/ts/delete`;
+export const DELETE_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/ts/delete`;
 
-export const ADD_COMPONENT_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/add/components?device=`;
+export const ADD_COMPONENT_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/add/components?device=`;
```
Comment (Baalmart) [inline netmanager/src/config/urls/deviceRegistry.js:25]: components routes need to be removed, we no longer have this functionality in our codebase @Codebmk

**Review by Baalmart — COMMENTED** [review]
Hi @Codebmk , I have left a new small review comment.
Also:

1. the v2 versions for recall, maintain and deploy have been updated in staging -- PR has been merged.
2. The PR for `data-mgt/feeds` is also about to be merged in a few mins.
3. I am also curious as to where the data-mgt/feeds route is situated?

Thanks again.

**Comment by Codebmk** [comment]
> Hi @Codebmk , I have left a new small review comment. Also:
> 
> 1. the v2 versions for recall, maintain and deploy have been updated in staging -- PR has been merged.
> 2. The PR for `data-mgt/feeds` is also about to be merged in a few mins.
> 3. I am also curious as to where the data-mgt/feeds route is situated?
> 
> Thanks again.

@Baalmart what do you mean by `where is data-mgt/feeds` route situated? Do you mean in the frontend/functionality? One of the apis user data-mgt is run device tests

**Comment by Baalmart** [comment]
Yes @Codebmk where is it situated in the frontend functionality?

**Comment by Codebmk** [comment]
> Yes @Codebmk where is it situated in the frontend functionality?

`/data/feeds/transform/recent` is used for run device tests
`/data/channels/maintenance/add` seems to not be used anywhere

**Comment by Baalmart** [comment]
> > Yes @Codebmk where is it situated in the frontend functionality?
> 
> `/data/feeds/transform/recent` is used for run device tests `/data/channels/maintenance/add` seems to not be used anywhere

thanks. @Codebmk

File: netmanager/src/config/urls/deviceRegistry.js (hunk 1 above)
Comment (Codebmk) [reply netmanager/src/config/urls/deviceRegistry.js]: done

**Pull request merged**